<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0,maximum-scale=1">
		
		<title>Publications : Developmental Cancer Genomics @ CCRI</title>

		<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon; charset=binary">

		<!-- Loading third party fonts -->
		<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
		<script src="https://use.fontawesome.com/af700ea065.js"></script>

		<!-- Loading main css file -->
		<link rel="stylesheet" href="css/style.css">
		
		<!--[if lt IE 9]>
		<script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
		<script src="https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.min.js"></script>
		<![endif]-->
		<style type="text/css">
			.publist p {
				 margin-left:32.0pt;
				 text-indent:-32.0pt;
			}
			.publist p .fa-star {
				margin-left: 64px;
			}
		</style>
	</head>


	<body>
		
		<div class="site-content">
			<header class="site-header" data-bg-image="">
				<div class="container">
					<div class="header-bar">
						<a href="index.html" class="branding">
							<img src="images/logo.png" alt="" class="logo">
							<div class="logo-type">
								<h1 class="site-title">Developmental Cancer Genomics</h1>
								<small class="site-description">Taking apart childhood cancer <tt>bit</tt> by <tt>bit</tt></small>
							</div>
						</a>

						<nav class="main-navigation">
							<button class="menu-toggle"><i class="fa fa-bars"></i></button>
							<ul class="menu">
								<li class="menu-item"><a href="index.html"><i class="fa fa-home fa-lg"></i></a></li>
								<li class="menu-item"><a href="join.html">Join us!</a></li>
								<li class="menu-item"><a href="people.html">People</a></li>
								<li class="menu-item"><a href="research.html">Research</a></li>
								<li class="menu-item current-menu-item"><a href="pubs.html">Publications</a></li>
								<li class="menu-item"><a href="contact.html">Contact</a></li>
							</ul>
						</nav>

						<div class="mobile-navigation"></div>
					</div>
				</div>
			</header>

			<div class="page-head" data-bg-image="images/banner4.png">
				<div class="container">
					<h2 class="page-title">Publications &amp; Resources</h2>
				</div>
			</div>

			<main class="main-content">
				<div class="fullwidth-block">
					<div class="container">
						<a name="pubs"></a>
						
<div class="publist">

Full publication lists: <a href="https://scholar.google.at/citations?user=kvN5U0IAAAAJ">Google Scholar</a>, <a href="https://orcid.org/0000-0003-2452-4784">ORCID</a>


						<h2 style="margin-top:2em">Current preprints:</h2>

<p><span>Lazic
D, Kromp F, Kirr M, Mivalt F, Rifatbegovic F, <b>Halbritter F</b>, Bernkopf M,
Bileck A, Ussowicz M, Ambros IM, Ambros PF, Gerner C, Ladenstein R, Ostalecki
C, Taschner-Mandl S. Single-cell landscape of bone marrow metastases in human
neuroblastoma unraveled by deep multiplex imaging. <i>bioRxiv</i>. Published
online September 30, 2020:2020.09.30.321539. <a
href="https://doi.org/10.1101/2020.09.30.321539">https://doi.org/10.1101/2020.09.30.321539</a>
<i class="fa fa-star"></i> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE94035">Data</a></span></p>

<p>Grissenberger S, Sturtzel C, Wenninger-Weinzierl A, Scheuringer E, Bierbaumer L, Pascoal S, Tötzl M, Tomazou E, Delattre O, Surdez D, Kovar H, <b>Halbritter F</b>, Distel M. Automated compound testing in zebrafish xenografts identifies combined MCL-1 and BCL-XL inhibition to be effective against Ewing sarcoma. <i>bioRxiv</i>. Published online June 17, 2021:2021.06.17.448794. <a href="https://doi.org/10.1101/2021.06.17.448794">https://doi.org/10.1101/2021.06.17.448794</a>
</span></p>

<p><span>&nbsp;</span></p>

						<h2>Main
contributions (first, last, or alphabetical authorship):</h2>

<p><span>Santini
L*, <b>Halbritter F*</b>, Titz-Teixeira F, Suzuki T, Asami M, Ramesmayer J, Ma
X, Lackner A, Warr N, Pauler F, Hippenmeyer S, Laue E, Farlik M, Bock C, Beyer
A, Perry ACF, Leeb M. Novel imprints in mouse blastocysts are predominantly DNA
methylation independent. <i>Nat Commun</i>. 2021;12:3804. <a href="https://doi.org/10.1038/s41467-021-23510-4">https://doi.org/10.1038/s41467-021-23510-4</a>
<i class="fa fa-star"></i> <a href="https://github.com/cancerbits/santini2021_imprints">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE152106">Data</a> | <a href="https://www.ccri.at/2021/06/21/hot-off-the-press-a-new-role-for-histone-modifications-in-genomic-imprinting/">Press@CCRI</a>
</span></p>

<p><b><span>Halbritter
F*</span></b><span>,
Farlik M*, Schwentner R, Jug G, Fortelny N, Schnöller T, Pisa H, Schuster LC,
Reinprecht A, Czech T, Gojo J, Holter W, Minkov M, Bauer WM, Simonitsch-Klupp
I, Bock C<sup>#</sup>, Hutter C<sup>#</sup>. Epigenomics and single-cell
sequencing define a developmental hierarchy in langerhans cell histiocytosis. <i>Cancer
Discov</i>. 2019;9(10):1406-1421. <a
href="https://doi.org/10.1158/2159-8290.CD-19-0138">https://doi.org/10.1158/2159-8290.CD-19-0138</a>

<i class="fa fa-star"></i>  <a href="http://lch-hierarchy.computational-epigenetics.org/">Website</a> | <a href="https://github.com/cancerbits/lch_hierarchy">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133706">Data</a> | <a href="https://www.ccri.at/2019/07/25/an-unexpected-developmental-hierarchy-in-an-unusual-disease/">Press@CCRI</a>

</span>
</p>

<p><span>Barakat
TS*, <b>Halbritter F*</b>, Zhang M, Rendeiro AF, Perenthaler E, Bock C,
Chambers I. Functional Dissection of the Enhancer Repertoire in Human Embryonic
Stem Cells. <i>Cell Stem Cell</i>. 2018;23(2):276-288. <a
href="https://doi.org/10.1016/j.stem.2018.06.014">https://doi.org/10.1016/j.stem.2018.06.014</a>
<i class="fa fa-star"></i>  <a href="http://hesc-enhancers.computational-epigenetics.org/">Website</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99631">Data</a>
</span></p>

<p><span>Allison
TF, Andrews PW, Avior Y, Barbaric I, Benvenisty N, Bock C, Brehm J, Brüstle O, Damjanov I, Elefanty A, Felkner D, Gokhale PJ, <b>Halbritter F</b>, Healy LE, Hu TX, Knowles BB, Loring JF, Ludwig TE, Mayberry R, et al. Assessment of established techniques to determine developmental and malignant potential of human pluripotent stem cells. <i>Nat Commun</i>. 2018;9(1):1925. <a
href="https://doi.org/10.1038/s41467-018-04011-3">https://doi.org/10.1038/s41467-018-04011-3</a>
<i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97964">Data</a>
</span></span></p>

<p><span>Mass
E*, Ballesteros I*, Farlik M, <b>Halbritter F*</b>, Günther P*, Crozet L,
Jacome-Galarza CE, Händler K, Klughammer J, Kobayashi Y, Gomez-Perdiguero E,
Schultze JL, Beyer M<sup>#</sup>, Bock C<sup>#</sup>, Geissmann F<sup>#</sup>.
Specification of tissue-resident macrophages during organogenesis. <i>Science
(80- )</i>. 2016;353(6304):aaf4238. <a
href="https://doi.org/10.1126/science.aaf4238">https://doi.org/10.1126/science.aaf4238</a>
<i class="fa fa-star"></i>  <a href="http://macrophage-development.computational-epigenetics.org/">Website</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81774">Data</a> | <a href="https://cemm.at/news/n/how-macrop">Press@CeMM</a> | <a href="https://www.mskcc.org/news/origin-story-finding-macrophage-development-challenges-long-held-view?_subsite=research-ski">Press@MSKCC</a>
</span></p>

<p><span>Bock
C, <b>Halbritter F</b>, Carmona FJ, Tierling S, Datlinger P, Assenov Y,
Berdasco M, Bergmann AK, Booher K, Busato F, Campan M, Dahl C, Dahmcke CM, Diep
D, Fernández AF, Gerhauser C, Haake A, Heilmann K, Holcomb T, et al.
Quantitative comparison of DNA methylation assays for biomarker development and
clinical applications. <i>Nat Biotechnol</i>. 2016;34(7):726-737. <a
href="https://doi.org/10.1038/nbt.3605">https://doi.org/10.1038/nbt.3605</a>
<i class="fa fa-star"></i>  <a href="https://github.com/epigen/methylation-biomarker-benchmark">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77965">Data</a> | <a href="https://cemm.at/news/n/epigenetic">Press@CeMM</a>
</span></p>

<p><span>Farlik
M*, <b>Halbritter F*</b>, Müller F*, Choudry FA, Ebert P, Klughammer J, Farrow
S, Santoro A, Ciaurro V, Mathur A, Uppal R, Stunnenberg HG, Ouwehand WH,
Laurenti E, Lengauer T, Frontini M<sup>#</sup>, Bock C<sup>#</sup>. DNA
Methylation Dynamics of Human Hematopoietic Stem Cell Differentiation. <i>Cell
Stem Cell</i>. 2016;19(6):808-822. <a
href="https://doi.org/10.1016/j.stem.2016.10.019">https://doi.org/10.1016/j.stem.2016.10.019</a>
<i class="fa fa-star"></i>  <a href="http://blueprint-methylomes.computational-epigenetics.org/">Website | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87196">Data</a> | <a href="https://cemm.at/news/n/beyond-the">Press@CeMM</a>
</span></p>

<p><b><span>Halbritter
F</span></b><span>,
Kousa AI, Tomlinson SR. </span><span>GeneProf
data: A resource of curated, integrated and reusable high-throughput genomics
experiments. <i>Nucleic Acids Res</i>. 2014;42(D1):D851-8. <a
href="https://doi.org/10.1093/nar/gkt966">https://doi.org/10.1093/nar/gkt966</a><i class="fa fa-star"></i>  <a href="http://www.geneprof.org/">Web service</a> (operated by University of Edinburgh)</p>

<p><b><span>Halbritter
F</span></b><span>,
Vaidya HJ, Tomlinson SR. GeneProf: Analysis of high-throughput sequencing
experiments. <i>Nat Methods</i>. 2012;9(1):7-8. <a
href="https://doi.org/10.1038/nmeth.1809">https://doi.org/10.1038/nmeth.1809</a><i class="fa fa-star"></i>  <a href="http://www.geneprof.org/">Web service</a> (operated by University of Edinburgh)</span></p>

<p><b><span>Halbritter F</span></b><span>,
Geibel P. Learning Models of Relational MDPs Using Graph Kernels. In: <i>MICAI
2007: Advances in Artificial Intelligence</i>. Springer; 2007:409-419. <a
href="https://doi.org/10.1007/978-3-540-76631-5_39">https://doi.org/10.1007/978-3-540-76631-5_39</a></span></p>

<p><span>&nbsp;</span></p>

<h2>Other contributions:</h2>

<p><span>Agerer
B*, Koblischke M*, Gudipati V*, <b>Montaño-Gutierrez LF</b>, Smyth M, Popa A,
Genger JW, Endler L, Florian DM, Mühlgrabner V, Graninger M, Aberle SW, Husa
AM, Shaw LE, Lercher A, Gattinger P, Torralba-Gombau R, Trapin D, Penz T, et
al. SARS-CoV-2 mutations in MHC-I-restricted epitopes evade CD8+ T cell
responses. <i>Sci Immunol</i>. 2021;6(57). <a
href="https://doi.org/10.1126/sciimmunol.abg6461">https://doi.org/10.1126/sciimmunol.abg6461</a><i class="fa fa-star"></i>  <a href="https://github.com/cancerbits/agerer2021_covid_epitopes">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE166651">Data</a>   </span></p>

<p><span>Thomas
HF, Kotova E, Jayaram S, Pilz A, Romeike M, Lackner A, Penz T, Bock C, Leeb M, <b>Halbritter
F</b>, Wysocka J, Buecker C. Temporal dissection of an enhancer cluster reveals
distinct temporal and functional contributions of individual elements. <i>Mol
Cell</i>. 2021;81(5):969-982.e13. <a
href="https://doi.org/10.1016/j.molcel.2020.12.047">https://doi.org/10.1016/j.molcel.2020.12.047</a></span></p>

<p><span>Cinquina
V, Calvigioni D, Farlik M, <b>Halbritter F</b>, Fife-Gernedl V, Shirran SL,
Fuszard MA, Botting CH, Poullet P, Piscitelli F, Máté Z, Szabó G, Yanagawa Y,
Kasper S, Di Marzo V, Mackie K, McBain CJ, Bock C, Keimpema E, et al. Life-long
epigenetic programming of cortical architecture by maternal ‘Western’ diet
during pregnancy. <i>Mol Psychiatry</i>. 2020;25(1):22-36. <a
href="https://doi.org/10.1038/s41380-019-0580-4">https://doi.org/10.1038/s41380-019-0580-4</a><i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140011">Data</a></span></p>

<p><span>Baumgartner
C, Toifl S, Farlik M, <b>Halbritter F</b>, Scheicher R, Fischer I, Sexl V, Bock
C, Baccarini M. An ERK-Dependent Feedback Mechanism Prevents Hematopoietic Stem
Cell Exhaustion. <i>Cell Stem Cell</i>. 2018;22(6):879-892. <a
href="https://doi.org/10.1016/j.stem.2018.05.003">https://doi.org/10.1016/j.stem.2018.05.003</a><i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112842">Data</a></span></p>

<p><span>Pham
HTT, Maurer B, Prchal-Murphy M, Grausenburger R, Grundschober E, Javaheri T,
Nivarthi H, Boersma A, Kolbe T, Elabd M, <b>Halbritter F</b>, Pencik J, Kazemi
Z, Grebien F, Hengstschläger M, Kenner L, Kubicek S, Farlik M, Bock C, et al.
STAT5B N642H is a driver mutation for T cell neoplasia. <i>J Clin Invest</i>.
2018;128(1):387-401. <a href="https://doi.org/10.1172/JCI94509">https://doi.org/10.1172/JCI94509</a></span></p>

<p><span>Festuccia
N, <b>Halbritter F</b>, Corsinotti A, Gagliardi A, Colby D, Tomlinson SR,
Chambers I. Esrrb extinction triggers dismantling of naïve pluripotency and
marks commitment to differentiation. <i>EMBO J</i>. 2018;37(21):e95476. <a
href="https://doi.org/10.15252/embj.201695476">https://doi.org/10.15252/embj.201695476</a>
<i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118907">Data</a>
</span></p>

<p><span>Corsinotti
A*, Wong FCK*, Tatar T, Szczerbinska I, <b>Halbritter F</b>, Colby D, Gogolok
S, Pantier R, Liggat K, Mirfazeli ES, Ponsele EH, Mullin NP, Wilson V, Chambers
I. Distinct SoxB1 networks are required for naïve and primed pluripotency. <i>Elife</i>.
2017;6:e27746. <a href="https://doi.org/10.7554/eLife.27746">https://doi.org/10.7554/eLife.27746</a>
<i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99185">Data</a></span></p>

<p><span>Mistri
TK, Devasia AG, Chu LT, Ng WP, <b>Halbritter F</b>, Colby D, Martynoga B,
Tomlinson SR, Chambers I, Robson P, Wohland T. Selective influence of Sox2 on
POU transcription factor binding in embryonic and neural stem cells. <i>EMBO
Rep</i>. 2015;16(9):1177-1191. <a href="https://doi.org/10.15252/embr.201540467">https://doi.org/10.15252/embr.201540467</a></span></p>

<p><span>Karwacki-Neisius
V, Göke J, Osorno R, <b>Halbritter F</b>, Ng JH, Weiße AY, Wong FCK, Gagliardi
A, Mullin NP, Festuccia N, Colby D, Tomlinson SR, Ng HH, Chambers I. Reduced
Oct4 expression directs a robust pluripotent state with distinct signaling
activity and increased enhancer occupancy by Oct4 and Nanog. <i>Cell Stem Cell</i>.
2013;12(5):531-545. <a href="https://doi.org/10.1016/j.stem.2013.04.023">https://doi.org/10.1016/j.stem.2013.04.023</a>
<i class="fa fa-star"></i>  <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1619/">Data</a></span></p>

<p><span>Gagliardi
A*, Mullin NP*, Ying Tan Z, Colby D, Kousa AI, <b>Halbritter F</b>, Weiss JT,
Felker A, Bezstarosti K, Favaro R, Demmers J, Nicolis SK, Tomlinson SR, Poot
RA, Chambers I. A direct physical interaction between Nanog and Sox2 regulates
embryonic stem cell self-renewal. </span><i><span>EMBO J</span></i><span>.
2013;32(16):2231-2247. <a href="https://doi.org/10.1038/emboj.2013.161"><span>https://doi.org/10.1038/emboj.2013.161</span></a></span></p>

<p><span>Festuccia
N*, Osorno R*, <b>Halbritter F</b>, Karwacki-Neisius V, Navarro P, Colby D,
Wong F, Yates A, Tomlinson SR, Chambers I. Esrrb is a direct Nanog target gene
that can substitute for Nanog function in pluripotent cells. <i>Cell Stem Cell</i>.
2012;11(4):477-490. <a href="https://doi.org/10.1016/j.stem.2012.08.002">https://doi.org/10.1016/j.stem.2012.08.002</a></span></p>

<p><span>Lee
EK*, Jin YW*, Park JH, Yoo YM, Hong SM, Amir R, Yan Z, Kwon E, Elfick A,
Tomlinson S, <b>Halbritter F</b>, Waibel T, Yun BW, Loake GJ. Cultured cambial
meristematic cells as a source of plant natural products. <i>Nat Biotechnol</i>.
2010;28(11):1213-1217. <a href="https://doi.org/10.1038/nbt.1693">https://doi.org/10.1038/nbt.1693</a></span></p>

<p><span>&nbsp;</span></p>

</div>
					</div>
				</div>
			</main> <!-- .main-content -->

			<footer class="site-footer">
				<div class="container">
					<hr/>
					<div class="row">
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">Our address</h3>
								<strong>Developmental Cancer Genomics / Integrative Analysis</strong>
								<div>Florian Halbritter Lab</div>
								<address>St. Anna Children's Cancer Research Institute (CCRI), Zimmermannplatz 10, 1090 Vienna, Austria</address>
								<span class="nairolf" data-user="rettirblah" data-website="ta.ircc">
							</div>
						</div>
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">Our host</h3>
								<a href="https://science.ccri.at/"><img src="images/logo-ccri.gif" alt="Children's Cancer Research Institute"/></a>
							</div>
						</div>
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">On the web</h3>
								<p>Other presence of the lab and members.</p>
								<div class="social-links">
									<a href="https://github.com/cancerbits"><i class="fa fa-github"></i></a>
									<a href="https://twitter.com/cancerbits"><i class="fa fa-twitter"></i></a>
									<a href="https://scholar.google.at/citations?user=kvN5U0IAAAAJ"><i class="fa fa-google"></i></a>
									<a href="https://www.linkedin.com/pub/florian-halbritter/70/79/4b7"><i class="fa fa-linkedin"></i></a>
									<a href="https://orcid.org/0000-0003-2452-4784">iD</a>
									<a href="https://www.researchgate.net/profile/Florian_Halbritter">RG</a>
								</div>
							</div>
						</div>
					</div> <!-- .row -->

					<p class="colophon">Copyright 2020 Florian Halbritter. Design template by <a href="http://www.vandelaydesign.com/" title="Designed by VandelayDesign.com" target="_blank">VandelayDesign.com</a>. All rights reserved.</p>
				</div> <!-- .container -->
			</footer>
		</div>

		<script type="text/javascript" src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
		<script type="text/javascript" src="js/plugins.js"></script>
		<script type="text/javascript" src="js/app.js"></script>
		
	</body>

</html>
