<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0,maximum-scale=1">
		
		<title>Publications : Developmental Cancer Genomics @ CCRI</title>

		<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon; charset=binary">

		<!-- Loading third party fonts -->
		<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
		<script src="https://use.fontawesome.com/af700ea065.js"></script>

		<!-- Loading main css file -->
		<link rel="stylesheet" href="css/style.css">
		
		<!--[if lt IE 9]>
		<script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
		<script src="https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.min.js"></script>
		<![endif]-->
		<style type="text/css">
			.publist p {
				 margin-left:32.0pt;
				 text-indent:-32.0pt;
			}
			.publist p .fa-star {
				margin-left: 64px;
			}
		</style>
	</head>


	<body>
		<div class="site-content">
			<header class="site-header" data-bg-image="">
				<div class="container">
					<div class="header-bar">
						<a href="index.html" class="branding">
							<img src="images/logo.png" alt="" class="logo">
							<div class="logo-type">
								<h1 class="site-title">Developmental Cancer Genomics</h1>
								<small class="site-description">Taking apart childhood cancer <tt>bit</tt> by <tt>bit</tt></small>
							</div>
						</a>

						<nav class="main-navigation">
							<button class="menu-toggle"><i class="fa fa-bars"></i></button>
							<ul class="menu">
								<li class="menu-item"><a href="index.html"><i class="fa fa-home fa-lg"></i></a></li>
								<li class="menu-item"><a href="join.html">Join us!</a></li>
								<li class="menu-item"><a href="people.html">People</a></li>
								<li class="menu-item"><a href="research.html">Research &amp; Funding</a></li>
								<li class="menu-item current-menu-item"><a href="pubs.html">Publications</a></li>
								<li class="menu-item"><a href="contact.html">Contact</a></li>
							</ul>
						</nav>

						<div class="mobile-navigation"></div>
					</div>
				</div>
			</header>

			<div class="page-head" data-bg-image="images/banner4.png">
				<div class="container">
					<h2 class="page-title">Publications &amp; Resources</h2>
				</div>
			</div>

			<main class="main-content">
				<div class="fullwidth-block">
					<div class="container">
						<a name="pubs"></a>
						
<div class="publist">

<p>Full publication lists: <a href="https://scholar.google.at/citations?user=kvN5U0IAAAAJ">Google Scholar</a>, <a href="https://orcid.org/0000-0003-2452-4784">ORCID</a></p>

<p>Lab members <u>underlined</u>. *,# mark equal contributions.<p>

<h2 style="margin-top:2em">Current preprints:</h2>

<p>Saldana-Guerrero IM*, <u>Montano-Gutierrez LF*</u>, <u>Hafemeister C</u>, Stavish D, Shaw LE, Fetahu IS, Wenninger-Weinzierl A, Sturtzel C, Souilhol C, Tarelli S, <u>Shoeb MR</u>, Bernkopf M, Bozatzi P, Guarini M, Bozsaky E, Buri MC, Putz EM, Andrews PW, Barbaric I, Bryant HE, Distel M, Taschner-Mandl S, Farlik M, Tsakiridis A<sup>#</sup>, <u>Halbritter F<sup>#</sup></u>. Neuroblastoma-associated chromosomal aberrations drive cell identity loss in human neural crest via disruption of developmental regulators. <b>bioRxiv</b>. 2022:2022.11.21.515753. <a href="https://doi.org/10.1101/2022.11.21.515753">https://doi.org/10.1101/2022.11.21.515753</a></p>
<p>Kirchberger S*, <u>Shoeb MR*</u>, Lazic D, Fischer K, Shaw LE, Nogueira F, Rifatbegovic F, Bozsaky E, Ladenstein R, Bodenmiller B, Lion T, Traver D, Farlik M, Taschner-Mandl S, <u>Halbritter F<sup>#</sup></u>, Distel M<sup>#</sup>. Cross-species analysis identifies conserved transcriptional mechanisms of neutrophil maturation. <b>bioRxiv</b>. 2022:2022.11.28.518146. <a href="https://doi.org/10.1101/2022.11.28.518146">https://doi.org/10.1101/2022.11.28.518146</a></p>
<p><u>Hafemeister C</u>, <u>Halbritter F</u>. Single-cell RNA-seq differential expression tests within a sample should use pseudo-bulk data of pseudo-replicates. <b>bioRxiv</b>. 2023:2023.03.28.534443. <a href="https://doi.org/10.1101/2023.03.28.534443">https://doi.org/10.1101/2023.03.28.534443</a></p>

	
<p>&nbsp;</p>

<h2>Main contributions (first | last authorship):</h2>

<p>Santini L*, <u>Halbritter F*</u>, Titz-Teixeira F, Suzuki T, Asami M, Ramesmayer J, Ma X, Lackner A, Warr N, Pauler F, Hippenmeyer S, Laue E, Farlik M, Bock C, Beyer A, Perry ACF, Leeb M. Novel imprints in mouse blastocysts are predominantly DNA methylation independent. <b>Nat Commun</b>. 2021;12:3804. <a href="https://doi.org/10.1038/s41467-021-23510-4">https://doi.org/10.1038/s41467-021-23510-4</a> <i class="fa fa-star"></i> <a href="https://github.com/cancerbits/santini2021_imprints">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE152106">Data: GSE152106</a> | <a href="https://www.ccri.at/2021/06/21/hot-off-the-press-a-new-role-for-histone-modifications-in-genomic-imprinting/">Press@CCRI</a> </p>

<p><u>Halbritter F*</u>, Farlik M*, Schwentner R, Jug G, Fortelny N, Schnöller T, Pisa H, Schuster LC, Reinprecht A, Czech T, Gojo J, Holter W, Minkov M, Bauer WM, Simonitsch-Klupp I, Bock C<sup>#</sup>, Hutter C<sup>#</sup>. Epigenomics and single-cell sequencing define a developmental hierarchy in langerhans cell histiocytosis. <b>Cancer
Discov</b>. 2019;9(10):1406-1421. <a href="https://doi.org/10.1158/2159-8290.CD-19-0138">https://doi.org/10.1158/2159-8290.CD-19-0138</a> <i class="fa fa-star"></i>  <a href="http://lch-hierarchy.computational-epigenetics.org/">Website</a> | <a href="https://github.com/cancerbits/lch_hierarchy">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133706">Data: GSE133706</a> | <a href="https://www.ccri.at/2019/07/25/an-unexpected-developmental-hierarchy-in-an-unusual-disease/">Press@CCRI</a> </p>

<p>Barakat TS*, <u>Halbritter F*</u>, Zhang M, Rendeiro AF, Perenthaler E, Bock C, Chambers I. Functional Dissection of the Enhancer Repertoire in Human Embryonic Stem Cells. <b>Cell Stem Cell</b>. 2018;23(2):276-288. <a href="https://doi.org/10.1016/j.stem.2018.06.014">https://doi.org/10.1016/j.stem.2018.06.014</a> <i class="fa fa-star"></i>  <a href="http://hesc-enhancers.computational-epigenetics.org/">Website</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99631">Data: GSE99631</a> </p>

<p>Allison TF, Andrews PW, Avior Y, Barbaric I, Benvenisty N, Bock C, Brehm J, Brüstle O, Damjanov I, Elefanty A, Felkner D, Gokhale PJ, <u>Halbritter F</u>, Healy LE, Hu TX, Knowles BB, Loring JF, Ludwig TE, Mayberry R, et al. Assessment of established techniques to determine developmental and malignant potential of human pluripotent stem cells. <b>Nat Commun</b>. 2018;9(1):1925. <a href="https://doi.org/10.1038/s41467-018-04011-3">https://doi.org/10.1038/s41467-018-04011-3</a> <i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97964">Data: GSE97964</a> </p>

<p>Mass E*, Ballesteros I*, Farlik M, <u>Halbritter F*</u>, Günther P*, Crozet L, Jacome-Galarza CE, Händler K, Klughammer J, Kobayashi Y, Gomez-Perdiguero E, Schultze JL, Beyer M<sup>#</sup>, Bock C<sup>#</sup>, Geissmann F<sup>#</sup>. Specification of tissue-resident macrophages during organogenesis. <b>Science (80- )</b>. 2016;353(6304):aaf4238. <a href="https://doi.org/10.1126/science.aaf4238">https://doi.org/10.1126/science.aaf4238</a> <i class="fa fa-star"></i>  <a href="http://macrophage-development.computational-epigenetics.org/">Website</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81774">Data: GSE81774</a> | <a href="https://cemm.at/news/n/how-macrop">Press@CeMM</a> | <a href="https://www.mskcc.org/news/origin-story-finding-macrophage-development-challenges-long-held-view?_subsite=research-ski">Press@MSKCC</a> </p>

<p>Bock C, <u>Halbritter F</u>, Carmona FJ, Tierling S, Datlinger P, Assenov Y, Berdasco M, Bergmann AK, Booher K, Busato F, Campan M, Dahl C, Dahmcke CM, Diep D, Fernández AF, Gerhauser C, Haake A, Heilmann K, Holcomb T, et al. Quantitative comparison of DNA methylation assays for biomarker development and clinical applications. <b>Nat Biotechnol</b>. 2016;34(7):726-737. <a href="https://doi.org/10.1038/nbt.3605">https://doi.org/10.1038/nbt.3605</a> <i class="fa fa-star"></i>  <a href="https://github.com/epigen/methylation-biomarker-benchmark">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77965">Data: GSE77965</a> | <a href="https://cemm.at/news/n/epigenetic">Press@CeMM</a> </p>

<p>Farlik M*, <u>Halbritter F*</u>, Müller F*, Choudry FA, Ebert P, Klughammer J, Farrow S, Santoro A, Ciaurro V, Mathur A, Uppal R, Stunnenberg HG, Ouwehand WH, Laurenti E, Lengauer T, Frontini M<sup>#</sup>, Bock C<sup>#</sup>. DNA Methylation Dynamics of Human Hematopoietic Stem Cell Differentiation. <b>Cell Stem Cell</b>. 2016;19(6):808-822. <a href="https://doi.org/10.1016/j.stem.2016.10.019">https://doi.org/10.1016/j.stem.2016.10.019</a> <i class="fa fa-star"></i>  <a href="http://blueprint-methylomes.computational-epigenetics.org/">Website | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87196">Data: GSE87196</a> | <a href="https://cemm.at/news/n/beyond-the">Press@CeMM</a> </p>

<p><u>Halbritter F</u>, Kousa AI, Tomlinson SR. GeneProf data: A resource of curated, integrated and reusable high-throughput genomics experiments. <b>Nucleic Acids Res</b>. 2014;42(D1):D851-8. <a
href="https://doi.org/10.1093/nar/gkt966">https://doi.org/10.1093/nar/gkt966</a><i class="fa fa-star"></i>  <a href="http://www.geneprof.org/">Web service</a> (operated by University of Edinburgh)</p>

<p><u>Halbritter F</u>, Vaidya HJ, Tomlinson SR. GeneProf: Analysis of high-throughput sequencing experiments. <b>Nat Methods</b>. 2012;9(1):7-8. <a href="https://doi.org/10.1038/nmeth.1809">https://doi.org/10.1038/nmeth.1809</a><i class="fa fa-star"></i>  <a href="http://www.geneprof.org/">Web service</a> (operated by University of Edinburgh)</p>

<p><u>Halbritter F</u>, Geibel P. Learning Models of Relational MDPs Using Graph Kernels. In: <b>MICAI 2007: Advances in Artificial Intelligence</b>. Springer; 2007:409-419. <a href="https://doi.org/10.1007/978-3-540-76631-5_39">https://doi.org/10.1007/978-3-540-76631-5_39</a></p>

<p>&nbsp;</p>


<h2>Other contributions:</h2>

<p> Kostel Bal S, Giuliani S, Block J, Repiscak P, <u>Hafemeister C</u>, Shahin T, Kasap N, Ransmayr B, Miao Y, van de Wetering C, Frohne A, Jimenez-Heredia R, Schuster MK, Zoghi S, Hertlein V, Thian M, Bykov A, Babayeva R, Eltan SB, Karakoc-Aydiner E, Shaw LE, Chowdury I, Varjosalo M, Argüello RJ, Farlik M, Ozen A, Serfling EAE, Dupré L, Bock C, <u>Halbritter F</u>, Hannich JT, Castanon I, Kraakman MJ, Baris S, Boztug K. Biallelic NFATC1 mutations cause an inborn error of immunity with impaired CD8+ T-cell function and perturbed glycolysis. <b>Blood</b>. 2023; blood.2022018303. <a href="https://doi.org/10.1182/blood.2022018303">https://doi.org/10.1182/blood.2022018303</a> <i class="fa fa-star"></i> <a href="https://github.com/cancerbits/kostelbal2023_nfatc1">Code</a></p>

<p>Grissenberger S, Sturtzel C, Wenninger-Weinzierl A, Radic-Sarikas B, Scheuringer E, Bierbaumer L, Etienne V, Némati F, Pascoal S, Tötzl M, Tomazou E, Metzelder M, Putz EM, Decaudin D, Delattre O, Surdez D, Kovar H, <u>Halbritter F</u>, Distel M. Automated compound testing in zebrafish xenografts identifies combined MCL-1 and BCL-XL inhibition to be effective against Ewing sarcoma. <b>Cancer Letters</b>. 2022; 554:216208. <a href="https://doi.org/10.1016/j.canlet.2022.216028">https://doi.org/10.1016/j.canlet.2022.216028</a></p>

<p>Gogolou A, Souilhol C, Granata I, Wymeersch FJ, Manipur I, Wind M, Frith TJ, Guarini M, Bertero A, Bock C, <u>Halbritter F</u>, Takasato M, Guarracino MR, Tsakiridis A. Early anteroposterior regionalisation of human neural crest is shaped by a pro-mesodermal factor. <b>eLife</b>. 2022; 11: e74263. <a href="https://doi.org/10.7554/eLife.74263">https://doi.org/10.7554/eLife.74263</a> <i class="fa fa-star"></i> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE184227">Data: GSE184227</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE184622">Data: GSE184622</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE184620">Data: GSE184620</a></p>

<p>Shahin T, Mayr D, <u>Shoeb MR</u>, Kuehn HS, Hoeger B, Giuliani S, Gawriyski L, Yüce Petronczki Ö, Hadjadj J, Kostel Bal S, Zoghi S, Haimel M, Jimenez-Heredia R, Boutboul D, Triebwasser MP, Rialland-Battisti F, Costedoat-Chalumeau N, Quartier P, Tangye SG, Fleisher TA, Rezaei N, Romberg N, Latour S, Varjosalo M, <u>Halbritter F</u>, Rieux-Laucat F, Castanon I, Rosenzweig S, Boztug K. Identification of Germline Monoallelic Mutations in IKZF2 in Patients with Immune Dysregulation. <b>Blood Adv</b>. 2021; bloodadvances.2021006367. <a
href="https://doi.org/10.1182/bloodadvances.2021006367">https://doi.org/10.1182/bloodadvances.2021006367</a> <i class="fa fa-star"></i> <a href="https://github.com/cancerbits/shahin2021_ikzf2_het">Code</a> | <a href="https://ega-archive.org/studies/EGAS00001005874">Data: EGAS00001005874</a> </p>

<p>Shahin T, Kuehn HS, <u>Shoeb MR</u>, Gawriyski L, Giuliani S, Repiscak P, Hoeger B, Yüce Petronczki Ö, Bal SK, Zoghi S, Dmytrus J, Seruggia D, Castanon I, Rezaei N, Varjosalo M, <u>Halbritter F</u>, Rosenzweig SD, Boztug K. Germline biallelic mutation affecting the transcription factor Helios causes pleiotropic defects of immunity. <b>Sci Immunol</b> 2021; 6(65):eabe3981. <a
href="https://doi.org/10.1126/sciimmunol.abe3981  ">https://doi.org/10.1126/sciimmunol.abe3981  </a> <i class="fa fa-star"></i> <a href="https://github.com/cancerbits/shahin2021_ikzf2">Code</a> | <a href="https://ega-archive.org/studies/EGAS00001005675">Data: EGAS00001005675</a> </p>

<p>Rindler K, Jonak C, Alkon N, Thaler FM, Kurz H, Shaw LE, Stingl G, Weninger W, <u>Halbritter F</u>, Bauer WM, Farlik M, Brunner PM. Single-cell RNA sequencing reveals markers of disease progression in primary cutaneous T-cell lymphoma. <b>Mol Cancer</b>. 2021 Sep 28;20(1):124. <a
href="https://doi.org/10.1186/s12943-021-01419-2">https://doi.org/10.1186/s12943-021-01419-2</a> <i class="fa fa-star"></i> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE173205">Data: GSE173205</a> </p>

<p>Eder SK, Schwentner R, Ben Soussia P, Abagnale G, Attarbaschi A, Minkov M, <u>Halbritter F</u>, Hutter C. Vemurafenib acts as molecular on-off switch governing systemic inflammation in Langerhans cell histiocytosis. <b>Blood Adv</b>. 2021 Oct 7:bloodadvances.2021005442. <a
href="https://doi.org/10.1182/bloodadvances.2021005442">https://doi.org/10.1182/bloodadvances.2021005442</a> <i class="fa fa-star"></i> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175480">Data: GSE175480</a> </p>

<p>Kollmann S, Grausenburger R, Klampfl T, Prchal-Murphy M, Bastl K, <u>Pisa H</u>, Knab VM, Brandstoetter T, Doma E, Sperr WR, Lagger S, Farlik M, Moriggl R, Valent P, <u>Halbritter F</u>, Kollmann K, Heller G, Maurer B, Sexl V. A STAT5B-CD9 axis determines self-renewal in hematopoietic and leukemic stem cells. <b>Blood</b>. 2021. <a
href="https://doi.org/10.1182/blood.2021010980">https://doi.org/10.1182/blood.2021010980</a> </p>

<p>Lazic D, Kromp F, Kirr M, Mivalt F, Rifatbegovic F, <u>Halbritter F</u>, Bernkopf M, Bileck A, Ussowicz M, Ambros IM, Ambros PF, Gerner C, Ladenstein R, Ostalecki C, Taschner-Mandl S. Single-cell landscape of bone marrow metastases in human neuroblastoma unraveled by deep multiplex imaging. <b>Cancers</b>. 2021; 13(17), 4311. <a
href="https://doi.org/10.3390/cancers13174311">https://doi.org/10.3390/cancers13174311</a> <i class="fa fa-star"></i> <a href="https://cloud.stanna.at/sharing/qiN0u9QPO">Data: Imaging</a> | <a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD018267">Data: PXD018267</a> | <a href="https://hub.docker.com/repository/docker/imageprocessing29092020/deepflex">Docker</a> | <a href="https://github.com/prepiscak/neuroblastoma_BM_infiltration">Code</a> | <a href="https://www.ccri.at/2021/09/02/bone-marrow-metastases-for-the-first-time-analyzed-cell-by-cell/">Press@CCRI</a></p>

<p>Agerer B*, Koblischke M*, Gudipati V*, <u>Montaño-Gutierrez LF</u>, Smyth M, Popa A, Genger JW, Endler L, Florian DM, Mühlgrabner V, Graninger M, Aberle SW, Husa AM, Shaw LE, Lercher A, Gattinger P, Torralba-Gombau R, Trapin D, Penz T, et al. SARS-CoV-2 mutations in MHC-I-restricted epitopes evade CD8+ T cell responses. <b>Sci Immunol</b>. 2021;6(57). <a href="https://doi.org/10.1126/sciimmunol.abg6461">https://doi.org/10.1126/sciimmunol.abg6461</a> <i class="fa fa-star"></i>  <a href="https://github.com/cancerbits/agerer2021_covid_epitopes">Code</a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE166651">Data: GSE166651</a>   </p>

<p>Thomas HF, Kotova E, Jayaram S, Pilz A, Romeike M, Lackner A, Penz T, Bock C, Leeb M, <u>Halbritter F</u>, Wysocka J, Buecker C. Temporal dissection of an enhancer cluster reveals distinct temporal and functional contributions of individual elements. <b>Mol Cell</b>. 2021;81(5):969-982.e13. <a href="https://doi.org/10.1016/j.molcel.2020.12.047">https://doi.org/10.1016/j.molcel.2020.12.047</a></p>

<p>Cinquina V, Calvigioni D, Farlik M, <u>Halbritter F</u>, Fife-Gernedl V, Shirran SL, Fuszard MA, Botting CH, Poullet P, Piscitelli F, Máté Z, Szabó G, Yanagawa Y, Kasper S, Di Marzo V, Mackie K, McBain CJ, Bock C, Keimpema E, et al. Life-long epigenetic programming of cortical architecture by maternal ‘Western’ diet during pregnancy. <b>Mol Psychiatry</b>. 2020;25(1):22-36. <a href="https://doi.org/10.1038/s41380-019-0580-4">https://doi.org/10.1038/s41380-019-0580-4</a><i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140011">Data: GSE140011</a></p>

<p>Baumgartner C, Toifl S, Farlik M, <u>Halbritter F</u>, Scheicher R, Fischer I, Sexl V, Bock C, Baccarini M. An ERK-Dependent Feedback Mechanism Prevents Hematopoietic Stem Cell Exhaustion. <b>Cell Stem Cell</b>. 2018;22(6):879-892. <a href="https://doi.org/10.1016/j.stem.2018.05.003">https://doi.org/10.1016/j.stem.2018.05.003</a><i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112842">Data: GSE112842</a></p>

<p>Pham HTT, Maurer B, Prchal-Murphy M, Grausenburger R, Grundschober E, Javaheri T, Nivarthi H, Boersma A, Kolbe T, Elabd M, <u>Halbritter F</u>, Pencik J, Kazemi
Z, Grebien F, Hengstschläger M, Kenner L, Kubicek S, Farlik M, Bock C, et al. STAT5B N642H is a driver mutation for T cell neoplasia. <b>J Clin Invest</b>. 2018;128(1):387-401. <a href="https://doi.org/10.1172/JCI94509">https://doi.org/10.1172/JCI94509</a></p>

<p>Festuccia N, <u>Halbritter F</u>, Corsinotti A, Gagliardi A, Colby D, Tomlinson SR, Chambers I. Esrrb extinction triggers dismantling of naïve pluripotency and marks commitment to differentiation. <b>EMBO J</b>. 2018;37(21):e95476. <a href="https://doi.org/10.15252/embj.201695476">https://doi.org/10.15252/embj.201695476</a>
<i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118907">Data: GSE118907</a> </p>

<p>Corsinotti A*, Wong FCK*, Tatar T, Szczerbinska I, <u>Halbritter F</u>, Colby D, Gogolok S, Pantier R, Liggat K, Mirfazeli ES, Ponsele EH, Mullin NP, Wilson V, Chambers
I. Distinct SoxB1 networks are required for naïve and primed pluripotency. <b>Elife</b>. 2017;6:e27746. <a href="https://doi.org/10.7554/eLife.27746">https://doi.org/10.7554/eLife.27746</a>
<i class="fa fa-star"></i>  <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99185">Data: GSE99185</a></p>

<p>Mistri TK, Devasia AG, Chu LT, Ng WP, <u>Halbritter F</u>, Colby D, Martynoga B, Tomlinson SR, Chambers I, Robson P, Wohland T. Selective influence of Sox2 on POU transcription factor binding in embryonic and neural stem cells. <b>EMBO Rep</b>. 2015;16(9):1177-1191. <a href="https://doi.org/10.15252/embr.201540467">https://doi.org/10.15252/embr.201540467</a></p>

<p>Karwacki-Neisius V, Göke J, Osorno R, <u>Halbritter F</u>, Ng JH, Weiße AY, Wong FCK, Gagliardi A, Mullin NP, Festuccia N, Colby D, Tomlinson SR, Ng HH, Chambers I. Reduced Oct4 expression directs a robust pluripotent state with distinct signaling activity and increased enhancer occupancy by Oct4 and Nanog. <b>Cell Stem Cell</b>.
2013;12(5):531-545. <a href="https://doi.org/10.1016/j.stem.2013.04.023">https://doi.org/10.1016/j.stem.2013.04.023</a>
<i class="fa fa-star"></i>  <a href="https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1619/">Data: E-MTAB-1619</a></p>

<p>Gagliardi A*, Mullin NP*, Ying Tan Z, Colby D, Kousa AI, <u>Halbritter F</u>, Weiss JT, Felker A, Bezstarosti K, Favaro R, Demmers J, Nicolis SK, Tomlinson SR, Poot RA, Chambers I. A direct physical interaction between Nanog and Sox2 regulates embryonic stem cell self-renewal.<b>EMBO J</b>. 2013;32(16):2231-2247. <a href="https://doi.org/10.1038/emboj.2013.161">https://doi.org/10.1038/emboj.2013.161</a></p>

<p>Festuccia N*, Osorno R*, <u>Halbritter F</u>, Karwacki-Neisius V, Navarro P, Colby D, Wong F, Yates A, Tomlinson SR, Chambers I. Esrrb is a direct Nanog target gene
that can substitute for Nanog function in pluripotent cells. <b>Cell Stem Cell</b>. 2012;11(4):477-490. <a href="https://doi.org/10.1016/j.stem.2012.08.002">https://doi.org/10.1016/j.stem.2012.08.002</a></p>

<p>Lee EK*, Jin YW*, Park JH, Yoo YM, Hong SM, Amir R, Yan Z, Kwon E, Elfick A, Tomlinson S, <u>Halbritter F</u>, Waibel T, Yun BW, Loake GJ. Cultured cambial meristematic cells as a source of plant natural products. <b>Nat Biotechnol</b>. 2010;28(11):1213-1217. <a href="https://doi.org/10.1038/nbt.1693">https://doi.org/10.1038/nbt.1693</a></p>

<p>&nbsp;</p>

</div>
					</div>
				</div>
			</main> <!-- .main-content -->

			<footer class="site-footer">
				<div class="container">
					<hr/>
					<div class="row">
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">Our address</h3>
								<strong>Developmental Cancer Genomics / Integrative Analysis</strong>
								<div>Florian Halbritter Lab</div>
								<address>St. Anna Children's Cancer Research Institute (CCRI), Zimmermannplatz 10, 1090 Vienna, Austria</address>
								<span class="nairolf" data-user="rettirblah" data-website="ta.ircc">
							</div>
						</div>
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">Our host</h3>
								<a href="https://science.ccri.at/"><img src="images/logo-ccri.gif" alt="Children's Cancer Research Institute"/></a>
							</div>
						</div>
						<div class="col-md-4">
							<div class="widget">
								<h3 class="widget-title">On the web</h3>
								<p>Other presence of the lab and members.</p>
								<div class="social-links">
									<a href="https://github.com/cancerbits"><i class="fa fa-github"></i></a>
									<a href="https://twitter.com/cancerbits"><i class="fa fa-twitter"></i></a>
									<a href="https://scholar.google.at/citations?user=kvN5U0IAAAAJ"><i class="fa fa-google"></i></a>
									<a href="https://www.linkedin.com/pub/florian-halbritter/70/79/4b7"><i class="fa fa-linkedin"></i></a>
									<a href="https://orcid.org/0000-0003-2452-4784">iD</a>
									<a href="https://www.researchgate.net/profile/Florian_Halbritter">RG</a>
								</div>
							</div>
						</div>
					</div> <!-- .row -->

					<p class="colophon">Copyright 2020 Florian Halbritter. Design template by <a href="http://www.vandelaydesign.com/" title="Designed by VandelayDesign.com" target="_blank">VandelayDesign.com</a>. All rights reserved.</p>
				</div> <!-- .container -->
			</footer>
		</div>

		<script type="text/javascript" src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
		<script type="text/javascript" src="js/plugins.js"></script>
		<script type="text/javascript" src="js/app.js"></script>
		
	</body>

</html>
